### DISCUSSION TITLE
GSoC 2023 Proposal - Distinguishing real and artificial information

### DISCUSSION THREAD

**Discussion by Ishaanj18** [observingClouds/xbitinfo#181] [General]
Respected, @aaronspring @observingClouds 

I wanted to follow up with you and let you know that I have just sent you an email containing a copy of my proposal. I hope that it has reached you successfully and that you will have a chance to review it soon.

I am really excited about this project and I value your feedback and guidance as I move forward with it. I believe that your insight and experience will be extremely valuable to me, and I appreciate all of the time and effort that you have put into helping me so far.

If you have any questions or concerns about the proposal, I am eager to hear your thoughts and to discuss any aspects of the project that you feel could be improved.

 I look forward to hearing back from you soon.

Best regards,

Ishaan Jain

**Comment by observingClouds** [comment]
Thank you @Ishaanj18 very much for your proposal! We will contact you soon!
cc @milankl as he will be a mentor as well
